ark/cli/build.py

```diff
@@ -73,16 +73,20 @@ def build_site():
     if os.path.exists(site.res()):
         utils.copydir(site.res(), site.out())
 
-    # Callback to handle individual nodes. The `build_node` filter can be used
-    # as a switch to decide if a node should be written to disk. A `disable`
-    # flag in a node's metadata header will also prevent Ark from producing an
-    # output HTML page for a node.
-    def build_node(node):
-        if filters.apply('build_node', True, node) and not node.get('disable'):
+    # This callback writes an individual node to disk.
+    def write_node(node):
+        # A `disable` flag in a node's metadata header will prevent Ark from
+        # producing an output HTML page for the node.
+        if node.get('disable'):
```

Comment:
but this can't be used for (e.g.) directories, right? because nowhere to put the metadata

Reply:
Correct. The filter is the general mechanism.